_posts/en/newsletters/2021-02-24-newsletter.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Bitcoin Optech Newsletter #137'
+permalink: /en/newsletters/2021/02/24/
+name: 2021-02-24-newsletter
+slug: 2021-02-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the results of discussion about
+choosing activation parameters for a taproot soft fork and includes our
+regular sections with selected questions and answers from the Bitcoin
+StackExchange, releases and release candidates, and notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **Taproot activation discussion:** Michael Folkson [summarized][folkson
+  lot] a second meeting about activation parameters for [taproot][topic
+  taproot], concluding that the meeting was "largely unproductive" about
```

Comment:
"largely unproductive" were Luke's words quoted in my mailing list post. I wouldn't use these words. A lot of people including Core contributors declared their views which was helpful. Many people seemed to have made their minds up before the meeting though and there was little substantive discussion of the arguments for lot=true and lot=false during the meeting.